src/components/form/Dropdown.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="ff-dropdown" :class="'ff-dropdown--' + (isOpen ? 'open' : 'closed')" :disabled="disabled">
-        <div v-if="dropdownStyle === 'select'" @click="open()" class="ff-dropdown-selected">
+        <div v-if="dropdownStyle === 'select'" ref="dropdownLabel" class="ff-dropdown-selected" tabindex="0" @click="open()" @keydown.space.stop.prevent="open()">
```

Comment:
> @keydown.space.stop.prevent="open()

what am I missing here, what's the purpose of this?

Reply:
Without it, the `space` bubbles up to the DOM and scrolls down the page, moving the dropdown out of view. For reasons I didn't get to the bottom of, it needed _both_ stop and prevent to capture the event.